pkg/capi/components/CCVariables/Variable.vue

```diff
@@ -14,9 +14,10 @@ import { mapGetters } from 'vuex';
 import { Translation } from '@rancher/shell/types/t';
 import type { ClusterClassVariable } from '../../types/clusterClass';
 import { isDefined, openAPIV3SchemaValidators } from '../../util/validators';
-export default Vue.extend({
-  name: 'CCVariable',
 
+export default defineComponent({
+  name: 'CCVariable',
+  emits: ['validation-passed', 'update:value', 'input'],
```

Comment:
This seems to work as-is so I am being a little nitpicky, but it looks like the Variable component only emits `input` -- should that component be updated to use `update:value`?